docs/antrea-network-policy.md

```diff
@@ -248,10 +282,12 @@ selected cluster-wide using `podSelector`. If set with a `namespaceSelector`,
 all Pods from Namespaces selected by the namespaceSelector will be selected.
 Specific Pods from specific Namespaces can be selected by providing both a
 `podSelector` and a `namespaceSelector` in the same `appliedTo` entry.
+The `appliedTo` field can also reference a ClusterGroup resource by setting the ClusterGroup's name in `group` field in place of the stand-alone selectors.
```

Comment:
Would you add a new line?

Reply:
done

Reply:
by mistake amended commit instead of adding new commit